### PR TITLE
Revise default issue blueprint and mapping

### PR DESCRIPTION
**Pull request by phalbert**
Complement to https://github.com/port-labs/port-docs/pull/1083https://github.com/port-labs/port-docs/pull/1083

# Description

## What Changed:

**Enhanced wizIssue blueprint**: I added new properties to directly store valuable vulnerability information.

## Why This Matters:

- **User Value:** These changes provide Port users with a more streamlined Wiz integration experience.
- **Improved Visibility:** Users will now have faster access to:
  - The vulnerability itself
  - The affected cloud resource
  - The resource's owner
  - Remediation guidance

## How It Works:

- **Vulnerability Data:** The previous "vulnerability" object is replaced with specific properties within the wizIssue blueprint.

## Type of change

Please leave one option from the following and delete the rest:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] New Integration (non-breaking change which adds a new integration)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Non-breaking change (fix of existing functionality that will not change current behavior)
- [ ] Documentation (added/updated documentation)

## Screenshots

Include screenshots from your environment showing how the resources of the integration will look.

## API Documentation

Provide links to the API documentation used for this integration.

### CODE REVIEW

File: integrations/wiz/main.py [hunk 1]
```diff
@@ -38,6 +38,7 @@ async def on_resync(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
         async for _issues in wiz_client.get_issues():
             logger.info(f"Received {len(_issues)} issues")
             if kind == ObjectKind.ISSUE:
+                logger.info(_issues[0:5])
```
Comment (shalev007) [inline integrations/wiz/main.py:41]: Is this for developing?

File: integrations/wiz/pyproject.toml [hunk 2]
```diff
@@ -6,7 +6,7 @@ authors = ["Albert Luganga <albertluganga@getport.io>"]
 
 [tool.poetry.dependencies]
 python = "^3.11"
-port_ocean = {version = "^0.5.12", extras = ["cli"]}
+port_ocean = {version = "^0.5.13", extras = ["cli"]}
```
Comment (shalev007) [inline integrations/wiz/pyproject.toml:9]: Nice

File: integrations/wiz/main.py (hunk 1 above)
Comment (phalbert) [reply integrations/wiz/main.py:41]: Oh, yes. Let me remove that

**Review by shalev007 — APPROVED** [review]

File: integrations/wiz/main.py (hunk 1 above)
Comment (phalbert) [reply integrations/wiz/main.py]: resolved now

File: integrations/wiz/CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 <!-- towncrier release notes start -->
+# 0.1.18 (2024-05-01)
+
+### Improvements
+
+- Enhanced wizIssue blueprint with properties for vulnerability details (0.1.14)
```
Comment (MPTG94) [inline integrations/wiz/CHANGELOG.md:13]:
```suggestion
- Enhanced wizIssue blueprint with properties for vulnerability details
```

**Review by MPTG94 — APPROVED** [review]
Resolve my comment and I'll merge

**Pull request merged**